rmgpy/rmg/model.py

```diff
@@ -908,7 +908,7 @@ def generate_kinetics(self, reaction):
         G298 = reaction.get_free_energy_of_reaction(298)
         gibbs_is_positive = G298 > -1e-8
 
-        if family.own_reverse and hasattr(reaction, 'reverse'):
+        if family.own_reverse and len(reaction.products)==len(reaction.reactants) and hasattr(reaction, 'reverse'):
```

Comment:
I don't quite get why this helps... Can you please help me understand?

Reply:
So because of the decay framework we can now have reactions from a family that is it's own reverse where Nproducts > Nreactants. In these cases we can't use reverse kinetics so we don't want RMG to try to use them.